code/modules/client/preferences/README.md

```diff
@@ -363,30 +363,6 @@ Middleware can inject its own data at several points, such as providing new UI a
 
 ---
 
-## Jobs
-
-Every job must have an associated `.ts` file so that the UI knows where to place it.
-
-Create a file in `tgui/packages/tgui/interfaces/PreferencesMenu/jobs/jobs/`.
-
-This will specify the description and department of the job. Here is the details for the medical doctor:
```

Comment:
Never even caught that this was the cook and not the medical doctor

I just didnt want to show the backslash thing